src/Microsoft.PowerShell.Commands.Utility/commands/utility/Set-PSBreakpoint.cs

```diff
@@ -81,7 +81,7 @@ protected override void ProcessRecord()
                     else
```

Comment:
@rjmholt, your last commit had 7 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Breakpoint SDK Unit Tests.Managing breakpoints in the host runspace via the SDK.Can set command breakpoints
```powershell
Cannot find an overload for "SetCommandBreakpoint" and the argument count: "1".
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 44
44:             $Host.Runspace.Debugger.SetCommandBreakpoint('Test-ThisCommandDoesNotExist') | Should -BeOfType System.Management.Automation.CommandBreakpoint

```
Breakpoint SDK Unit Tests.Managing breakpoints in the host runspace via the SDK.Can set variable breakpoints
```powershell
Cannot find an overload for "SetVariableBreakpoint" and the argument count: "3".
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 48
48:             $Host.Runspace.Debugger.SetVariableBreakpoint('DebugPreference', 'ReadWrite', { continue }) | Should -BeOfType System.Management.Automation.VariableBreakpoint

```
Breakpoint SDK Unit Tests.Managing breakpoints in the host runspace via the SDK.Can get breakpoints
```powershell
Expected a collection with size 3, but got collection with size 1 @(Line breakpoint on '/home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1:1, 1').
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 56
56:             $Host.Runspace.Debugger.GetBreakpoints() | Should -HaveCount 3
```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can set command breakpoints
```powershell
Cannot find an overload for "SetCommandBreakpoint" and the argument count: "2".
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 104
104:             $jobRunspace.Debugger.SetCommandBreakpoint('Write-Verbose', { break }) | Should -BeOfType System.Management.Automation.CommandBreakpoint

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can set variable breakpoints
```powershell
Cannot find an overload for "SetVariableBreakpoint" and the argument count: "3".
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 108
108:             $jobRunspace.Debugger.SetVariableBreakpoint('DebugPreference', 'ReadWrite', { break }) | Should -BeOfType System.Management.Automation.VariableBreakpoint

```